packages/widgets/src/components/ExchangeAccount/styles.ts

```diff
@@ -0,0 +1,134 @@
+import { Theme, css } from '@emotion/react';
+
+export const CardWrapper = (theme: Theme) => css`
+  max-width: 500px;
+  padding: 0;
+  border-radius: 20px;
+  box-shadow: 0px 10px 40px 0px ${theme.colors.greyShadow};
+  transition: 0.3s;
+
+  &:hover {
+    box-shadow: 5px 5px 20px 0px ${theme.colors.black25};
+  }
+`;
+
+export const HeadWrapper = (theme: Theme) => css`
+  display: grid;
+  grid-template-columns: auto 1fr 20px;
+  align-items: center;
+  width: 100%;
+  padding: 10px;
+  border-radius: 20px 20px 0 0;
+  background: ${theme.colors.greyLighter};
+  margin-bottom: 0;
+
+  ${theme.mediaQueries.md} {
+    h6 {
+      font-size: 16px;
+      line-height: 24px;
+    }
+  }
+
+  ${theme.mediaQueries.xs} {
+    padding: 6px 10px;
+  }
+`;
+
+export const RemoveButton = (theme: Theme) => css`
+  height: auto;
+  padding: 0;
+  background: none;
+  box-shadow: none;
+
+  &:hover,
+  &:focus {
+    background: none;
+    box-shadow: none;
+  }
+
+  &:hover {
+    svg path {
+      fill: ${theme.colors.greyDarker};
+    }
+  }
+`;
+
+export const PlatformWrapper = css`
+  display: flex;
+  align-items: center;
+  gap: 9px;
+  margin-right: 13px;
+`;
+
+export const StatusWrapper = (theme: Theme) => css`
+  width: 100%;
+  text-align: center;
+  padding: 4px;
+
+  &.Active {
+    color: ${theme.colors.greenDark};
+    background: ${theme.colors.greenMint};
+  }
+
+  &.NotAvailable {
```

Comment:
I think, it's better to use `not-available` class name

Reply:
Done.